aries-site/src/components/cards/ContentCard.js

```diff
@@ -61,6 +83,11 @@ ContentCard.propTypes = {
       icon: PropTypes.func.isRequired,
       name: PropTypes.string.isRequired,
     }),
+    previewComponent: PropTypes.shape({
+      component: PropTypes.func,
+      cardColor: PropTypes.string,
+      justify: PropTypes.string,
+    }),
```

Comment:
Related to @taysea's https://github.com/hpe-design/design-system/pull/647#discussion_r405203370, I wonder if this becomes
```suggestion
    previewComponent: PropTypes.oneOfType([
      PropTypes.elementType,
      PropTypes.shape({
        component: PropTypes.func,
        cardColor: PropTypes.string,
        justify: PropTypes.string,
      })
    ]),
```